docs/site/submitting_a_pr.md

```diff
@@ -0,0 +1,495 @@
+---
+title: Submitting a pull request to LoopBack 4
+lang: en
+keywords: contributing, LoopBack community, pull request, PR, loopback
+sidebar: contrib_sidebar
+permalink: /doc/en/lb4/submitting_a_pr.html
+---
+
+## How to submit a pull request (PR) for LoopBack 4
+
+This tutorial is about how to submit a pull request (PR) for LoopBack 4 while
+following our conventions and requirements.
+
+Pick `Expert Instructions` or `Beginner Instructions` mode based your experience
+with GitHub's PR flow.
+
+To contribute high-quality code/docs, please read
+[Contributing to LoopBack](https://loopback.io/doc/en/contrib/index.html),
+[Contributing code](https://loopback.io/doc/en/contrib/code-contrib-lb4.html),
+and [Contributing to docs](https://loopback.io/doc/en/contrib/doc-contrib.html),
+before diving into the PR process.
+
+### Expert Instructions
+
+In addition to your knowledge about GitHub and creating a pull request, we have
+specific conventions and requirements you need to follow when submitting a pull
+request for LoopBack 4.
+
+#### 1. Commits to local repository
+
+Our commit messages are formatted according to
+[Conventional Commits](https://conventionalcommits.org/).
+
+Please read the
+[Commit Message Format](https://loopback.io/doc/en/contrib/code-contrib-lb4.html#commit-message-guidelines)
+guidelines to correctly format your commit messages.
+
+To change an existing commit please refer to
+[Changing a commit message](https://help.github.com/articles/changing-a-commit-message/).
+
+#### 2. Before pushing to remote repository
+
+Before pushing to the remote repository, ensure your files are free of
+formatting, syntax, and logical/execution errors, by running :
+
+```
+npm run lint:fix && npm run test
+```
+
+#### 3. After creating PR, sign the CLA and fill out checklist
+
+After creating the pull request, sign the Contributor License Agreement (CLA).
+
+Please read
+[Agreeing to the CLA](https://loopback.io/doc/en/contrib/code-contrib.html#agreeing-to-the-cla),
+click on this
+[link](https://cla.strongloop.com/agreements/strongloop/loopback-next), and sign
+it. This is done once per repository.
+
+The pull request has a checklist that must be filled out appropriately.
+
+![submit_pr_template_1.png](./imgs/submit_pr_template_1.png)
+
+Of particular importance are the `'npm test passes on your machine'` and
+`'Sign the CLA (Contributor License Agreement)'` checkboxes.
+
+#### 4. Check CI status
+
+Ensure that the continuous integration (CI) jobs associated with your pull
+request complete successfully.
```

Comment:
It might take some time for CI jobs to be scheduled and completed.

Reply:
We should tell contributors to click on `Details` for failing items to check. It might be nice to have a screenshot of travis CI report.